src/writeData/subscriptions/context/api.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  getBrokerSub,
+  GetBrokerSubParams,
+  getBrokerSubs,
+  GetBrokerSubsParams,
+  postBrokerSub,
+  PostBrokerSubParams,
+  putBrokerSub,
+  PutBrokerSubParams,
+  deleteBrokerSub,
+  DeleteBrokerSubParams,
+  getBrokerSubsStatus,
+  GetBrokerSubsStatusParams,
+} from 'src/client/nifidRoutes'
+
+export const createAPI = async (subscription: PostBrokerSubParams) => {
+  const res = await postBrokerSub(subscription)
+  if (res.status != 201) {
+    throw new Error(res.data.message)
+  }
+}
+
+export const updateAPI = async (subscription: PutBrokerSubParams) => {
```

Comment:
I think we're missing `updateStatusAPI` for PUT /{id}/status

Reply:
good catch, added